src/test/java/com/andreamazzarella/contact_manager/ContactTest.java

```diff
@@ -0,0 +1,21 @@
+package com.andreamazzarella.contact_manager;
+
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+
+public class ContactTest {
+
+    @Test
+    public void knowsIfNameContains() {
+        HashMap<String, String> contactInfo = new HashMap<>();
+        contactInfo.put("name", "Andrea");
+        Contact testContact = new Contact(contactInfo);
+
+        boolean matchesSearchTerm = testContact.nameContains("And");
+
+        assertEquals(true, matchesSearchTerm);
```

Comment:
Contact should probably be a POJO (plain old java object) with just getters, so tests for the getters are probably unnecessary.